gxjobconfinit/types.py

```diff
@@ -79,4 +81,5 @@ def from_dict(**data: Unpack[CliConfigArgs]) -> "ConfigArgs":
             singularity_sudo=data.get("singularity_sudo", None),
             singularity_sudo_cmd=data.get("singularity_sudo_cmd", None),
             singularity_extra_volume=data.get("singularity_extra_volume", []),
+            tmp_dir=data.get("tmp_dir", []),
```

Comment:
```suggestion
            tmp_dir=data.get("tmp_dir", True),
```
We should probably set up tests and type checking in the CI ? This should be a type error ?